### DISCUSSION TITLE
LLama2 support ?

### DISCUSSION THREAD

**Discussion by XinyueZ** [iryna-kondr/scikit-llm#45] [Q&A]
LLama2 support ?

**Comment by OKUA1** [comment]
@XinyueZ, 

Technically Llama 2 7b and 13b can be used by sideloading the models into gpt4all (the underlying architecture seems to be identical to llama1).

You can find the instructions here: https://docs.gpt4all.io/gpt4all_chat.html#sideloading-any-ggml-model
Exemplary model repo: https://huggingface.co/TheBloke/Nous-Hermes-Llama2-GGML

Also, you would need to install the latest scikit-llm directly from git `pip install git+https://github.com/iryna-kondr/scikit-llm`.

Unfortunately 70b version is not supported for now.
